tests/cluster/tests/21-many-slot-migration.tcl

```diff
@@ -3,8 +3,11 @@
 source "../tests/includes/init-tests.tcl"
 source "../tests/includes/utils.tcl"
 
+# TODO: This test currently runs without replicas, as failovers (which may
+# happen on lower-end CI platforms) are still not handled properly by the
+# cluster during slot migration.
```

Comment:
do you have a reference to an issue that describes the the problem in more detail?